packages/zapp/console/src/components/Executions/ExecutionDetails/NodeExecutionTabs/index.tsx

```diff
@@ -76,7 +79,12 @@ export const NodeExecutionTabs: React.FC<{
     }
   }
 
-  const executionLabel = isMapTaskType(taskTemplate?.type) ? 'Map Execution' : 'Executions';
+  const executionLabel = isMapTaskType(taskTemplate?.type)
+    ? shouldShowTaskDetails
+      ? 'Execution'
```

Comment:
nit: Why do we use singular "Execution" here? It still can have more than attempt for child element...
no need to change - just curious :)

Reply:
In specs it was singular, so I thought I should make a separate title for them